PowerEditor/src/NppIO.cpp

```diff
@@ -169,6 +170,13 @@ BufferID Notepad_plus::doOpen(const generic_string& fileName, bool isRecursive,
 	generic_string targetFileName = fileName;
 	resolveLinkFile(targetFileName);
 
+	bool WindowsFileNamingIssue = false;
+
+	if (targetFileName.at(0) == '\\' && targetFileName.at(1) == '\\' && targetFileName.at(2) == '?' && targetFileName.at(3) == '\\')

```

Comment:
You detect the specific case but it's not a generic solution.

Reply:
I have made the alterations suggested by @xomx but have not submitted a new pull request yet as I am not certain how to handle the case of the " " at the end of the document.